cpp/tests/io/json_writer.cpp

```diff
@@ -542,7 +567,9 @@ TEST_F(JsonWriterTest, Unicode)
                            .lines(true)
                            .na_rep("null");
 
-  cudf::io::write_json(options_builder.build(), rmm::mr::get_current_device_resource());
+  cudf::io::write_json(options_builder.build(),
+                       cudf::test::get_default_stream(),
+                       rmm::mr::get_current_device_resource());
```

Comment:
can these just be 
```suggestion
  cudf::io::write_json(options_builder.build());
```
We have the same values for the defaults. Not sure why `mr` was specified in the first place.

Reply:
now I see the difference, cudf::get_default_stream vs cudf::test::get_default_stream
does this matter for these tests? I don't see other PRs adding the stream parameter to all tests

Reply:
Yes, the stream parameter can be left as is for now. For the moment, `cudf::test::get_default_stream` is only necessary for the stream tests. That may change in the future, but probably isn't worth planning for right now.

Reply:
Thanks for the feedback! I've cleaned up the `rmm::mr` default parameter in both the calls.

Reply:
Not sure I understand what "as is" is in @vyasr 's comment, but it looks like we might be adding the stream parameter to all API calls in the future so we might as do it now.

FWIW, the `mr` parameter can be removed from `write_json` calls in this file as well. I'm not blocking the PR on that, though.